### PR TITLE
stop bridge after receiving SIGINT event

### PR DESCRIPTION
**Pull request by matejdro**
Migration from https://github.com/Sorunome/mx-puppet-slack/pull/70

### CODE REVIEW

File: src/puppetbridge.ts [hunk 1]
```diff
@@ -331,6 +331,12 @@ export class PuppetBridge extends EventEmitter {
 	public async start(callback?: () => Promise<void>) {
 		log.info("Starting application service....");
 
+		const AS = this.AS
+		process.on('SIGINT', function () {
+			AS.stop()
```
Comment (Sorunome) [inline src/puppetbridge.ts]: please use an arrow function and then use this.AS instead. Also, check the linter, missing semicolumns and it requires to use doublequotes ^^

```
process.on("SIGINT", () => {
    this.AS.stop();
// etc.
});
```

**Review by Sorunome — CHANGES_REQUESTED** [review]
Heehee, looking nice, just wanting arrow functions + `npm run lint`. Thanks a lot for the PR!

File: src/puppetbridge.ts (hunk 1 above)
Comment (matejdro) [reply src/puppetbridge.ts]: Updated.

**Comment by Sorunome** [comment]
Sorry it took so long to get back to this - Thanks a lot for the PR!

**Pull request merged**